src/Integration/LocalServices/ErrorListInfoBarController.cs

```diff
@@ -141,6 +137,11 @@ private void InvokeWhenIdle(Action action)
         }
 
         private void ClearCurrentInfoBar()
+        {
+            ClearCurrentInfoBarAsync().Forget();
```

Comment:
Clearing the info bar is still called from some synchronous code paths e.g. when the user clicks on a button. Those scenarios can safely be turned into "fire and forget".